packages/core/jsx-runtime/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@theme-ui/core-jsx-runtime",
+  "source": "../src/jsx-runtime.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
+  "dependencies": {
+    "@emotion/core": "^10.1.1",
+    "@theme-ui/parse-props": "^0.4.0-rc.5"
```

Comment:
The version here needs to keep in sync with the overall repo version. I can imagine that Lerna wouldn't update this one here because it isn't registered as workspace package. 🤔

Reply:
Random thought: Maybe we can change this line to `"@theme-ui/parse-props": "*"`?

Reply:
I changed it to `"@theme-ui/parse-props": "*"`, so we don't need to worry about that.